### PR TITLE
docs: Rewrite README to make it more appealing

### PR DESCRIPTION
**Pull request by aureleoules**
The current README is pretty bland and I believe it could need a few improvements to enhance its value and appeal. 

This pull request updates the README to make it more appealing. The changes are the following:
* Adding emojis 🚀 which makes the document more vibrant
* Use a more casual and interactive language to engage the reader

All the original technical contents were preserved, and the structure of the document remains the same to ensure that users still find the information they're accustomed to. 

These changes aim not only to keep the document detailed and informative but also to make it easy-to-read and engaging, especially to users who may be new to the Bitcoin Core project and attract new contributors.

See preview: https://github.com/aureleoules/bitcoin/blob/2023-07-update-readme/README.md.

### CODE REVIEW

**Review by jonatack — COMMENTED** [review]
A few suggestions.

(I'm not sure who the emojis and "We" narration would be more appealing to and which target audience is being aimed for, but no strong opinion. Sugarcoating things might be a little akin to false advertising, though 😄.)

**Comment by aureleoules** [comment]
Thanks @jonatack, I addressed your suggestions and rolled back some of my changes.

**Review by jonatack — COMMENTED** [review]
A few more suggestions. 

Just one opinion: in dark mode, which I use exclusively, I find the emojis are large, bright, and highly distracting from the text. If others agree, maybe remove them or make them smaller (not sure that's possible with markdown) or use fewer of them.

**Comment by russeree** [comment]
My technical objection is that emoji illustrations vary between browsers, devices, and browser versions. The use of emojis would create an inconsistent experience between platforms.

My emotional and personal objection is and I mean this in the nicest way possible. I don't like the 'vibe' the changes to more informal language creates. It also makes the document less concise. 

- The phrase ```Curious to know more? Find further details``` feels like a pitch almost as compared to the current ```Further information about Bitcoin Core is available```

- Use of an exclamation point in ```It downloads and validates transactions and blocks in real-time!``` feels forced and again like a pitch. Don't get me wrong I love me some freshie blocks and TXs but 

- Emojis make this feel like an AI model or application repo see https://github.com/Significant-Gravitas/Auto-GPT my preference is to have very concise unbiased language and visuals. Visuals should also not induce bias. This could be construed as good or bad.

**Comment by ariard** [comment]
I’m not sure keeping the project documentation as easy-to-read and engaging with emojis is that much a goal in itself considering Bitcoin Core is a security-first project underpinning a $567 billions ecosystem, where contributors would be rather invited to run their workflow on attack-surface minimized workstation. Rich graphical rendering requires lengthier and more opaque software supply chain.

Not a strong opinion, though there has been a [talk at DEFCON 30](https://media.defcon.org/DEF%20CON%2030/DEF%20CON%2030%20presentations/Hadrien%20Barral%20-%20Emoji%20Shellcoding.pdf) on how emojis can be used to craft shellcode :)

File: README.md [hunk 1]
```diff
@@ -1,71 +1,64 @@
...
 https://bitcoincore.org/en/download/.
 
-What is Bitcoin Core?
+What is Bitcoin Core? 🤔
 ---------------------
 
-Bitcoin Core connects to the Bitcoin peer-to-peer network to download and fully
-validate blocks and transactions. It also includes a wallet and graphical user
-interface, which can be optionally built.
+Bitcoin Core is your gateway to the Bitcoin network. It downloads and validates transactions and blocks in real-time! It also includes a wallet and graphical user interface (GUI) which can optionally be built.
```
Comment (stickies-v) [inline README.md:12]: I don't like this current rewording, but I think I agree with your intent of adding some more information of what the Bitcoin Core project stands for, e.g. highly reviewed, robust and stable code, one-stop-shop for running a node and wallet to fully interface with the Bitcoin Network, ...

**Review by stickies-v — COMMENTED** [review]
NACK. 

I think documentation needs to be as concise and simple as possible, but I feel like the current version does a good job at that already. The current language is factual and not more formal than it needs to be.

In a world of overmarketed crypto projects, I think maintaining sober documentation is a nice differentiation.

**Comment by aureleoules** [comment]
I understand the points you have raised and do somewhat agree with them. Closing this PR.

**Pull request closed without merging**